LibTrixi.jl/src/simulationstate.jl

```diff
@@ -1,12 +1,21 @@
+const LibTrixiDataBaseType = Vector{Ref{Vector{Float64}}}
```

Comment:
I wouldn't suffix a type with `Type` :-) Why not just `LibTrixiData{Base,Store,Whatever}`?

Reply:
Done. Now called `LibTrixiDataRegistry`.

Reply:
Nice!